### DISCUSSION TITLE
Προβλημα εγγραφης σε asciinema

### DISCUSSION THREAD

**Discussion by p16andr** [courses-ionio/sw#938] [Announcements]
Αντιμετωπιζω ενα προβλημα οταν παω να κανω εγγραφη στο asciinema απο τον τερματικο μου. Οταν κανω upload στον λογαριασμο μου στο asciinema το rec που εχω κανει δεν μου κανει το upload και μου πεταει το μηνυμα "Invalid or revoked install ID".  Υστερα απο ψαξιμο και απο βοηθεια συμφοιτητη μου ακολουθησα την παραπανω διαδικασια https://github.com/asciinema/asciinema/issues/272  αλλα και παλι δεν λυθηκε το θεμα. Αν καποιος εχει αντιμετωπισει το ιδιο θεμα και ξερει κατι περαιτερω θα με βοηθουσε παρα πολυ.

**Comment by ghost** [comment]
Εγώ θα σου πρότεινα, αν δεν το έχεις κάνει ήδη, να απεγκατασήσεις την εφαρμογή εντελώς και να την εγκαταστήσεις ξανά. Μετά κάνε login στο site του asciinema και κάνε μια καταγραφή για να ελέγξεις το αποτέλεσμα.

**Reply by p16andr** [reply]
@p18mili1 
Το δοκιμασα και αυτο και δυστυχως το προβλημα δεν λυθηκε. Μου εμφανιζει παλι το ιδιο error.

**Comment by elias2500** [comment]
Μπορείς να δοκιμάσεις να κάνεις καινούριο λογαριασμό στο Asciinema.

**Reply by p16andr** [reply]
@elias2500 
Ναι το εκανα και αυτο αλλα και παλι δεν γινεται τιποτα.

**Reply by elias2500** [reply]
Μπορείς να δοκιμάσεις και [αυτό](https://discourse.asciinema.org/t/why-upload-failed-invalid-or-revoked-install-id/27).

**Comment by apostoloskp** [comment]
Αν χρησιμοποιείς Ubuntu σε VM μπορείς να δοκιμάσεις να διαγράψεις ό,τι αρχείο υπάρχει αν κάνεις search **asciinema** στο Home folder και στη συνέχεια **+Other Locations**. Υπάρχει και η λύση να εγκαταστήσεις ξανά το iso από την αρχή αλλά είναι φασαρία.
